src/commands/selectSignImage.ts

```diff
@@ -0,0 +1,102 @@
+import * as vscode from 'vscode';
+import { logInToDockerCli } from './registries/logInToDockerCli';
+import { checkCLI } from '../utils/checkCLI';
+import { execAsync } from '../utils/execAsync';
+
+const cliName = 'notation';
+const goToNotationButton = 'Download Notation';
+const NotationURL = 'https://notaryproject.dev/docs/installation/cli/';
+const errorMessage = `Notation not found: `;
+
+//transforms the key list string into an array of objects
+async function sortOutput(Key_String_Values: string ){
```

Comment:
can we leave a future TODO comment here why this parsing is necessary? and that notation will plan to support JSON in the future instead for output?